server/controllers/routes/user/project/getProjectsData.js

```diff
@@ -0,0 +1,18 @@
+const {
+  getProjects,
+} = require('../../../../database/queries/project/getProjects');
+
+const getProjectsData = (req, res, next) => {
+  getProjects()
```

Comment:
@rehabas You could use await here, to make the code a little easier to read if you want:
```
const { rows } = getProjects();
res.json({
  StatusCode: 200,
  data: rows
})
```